app.py

```diff
@@ -1,18 +1,69 @@
-"""main app file for modeler"""
+#"""main app file for modeler"""
 def main():
     # just leaving these here for future reference (elmiko)
     # from pyspark import sql as pysql
     # spark = pysql.SparkSession.builder.appName("JiminyMod").getOrCreate()
     # sc = spark.sparkContext
-    while True:
+
+    import psycopg2
+    import pyspark
+    from pyspark.sql import SQLContext
+    from pyspark import SparkContext, SparkConf
+    import math
+
+    import modeller
+    import storage
+
+    ### set up the spark context.
+    conf = SparkConf().setAppName("recommender")
+    conf = (conf.setMaster('local[*]')
+            .set('spark.executor.memory', '4G')
+            .set('spark.driver.memory', '45G')
+            .set('spark.driver.maxResultSize', '10G'))
+    sc = SparkContext(conf=conf)
+    sqlContext=SQLContext(sc)
+    #Set up SQL Context
+    try:
+        con = psycopg2.connect(dbname='movielens', user='postgres', host='localhost', port='5432', password='password')
+    #    print "Connected to database"
+    except:
+        print "Cannot connect to the database"
```

Comment:
if we're going to catch the exception, we should probably just use a `return -1` here or even `sys.exit(1)` just to fail early without creating more errors.